application/usecases/driver/AddDriver.ts

```diff
@@ -0,0 +1,28 @@
+import type { DriverRepository } from "../../repositories/DriverRepository";
+import { DriverEntity } from "../../../domain/entities/DriverEntity";
+
+export class AddDriver {
+  public constructor(private readonly driverRepository: DriverRepository) {}
+
+  public async execute(
+    firstName: string,
+    lastName: string,
+    phone: string,
+    email: string,
+    birthDate: Date,
+    companyId: string,
+    DriverLicenseId: string
```

Comment:
Je pense que c'est une faute de frappe ?
```suggestion
    driverLicenseId: string
```